### ISSUE TITLE
AI problem asking to report issue

### ISSUE THREAD

**Issue by Neo2003** [ldtteam/minecolonies#4689]
### Minecolonies version

Version: 0.11.840-BETA

### Expected behavior

Builder to build
- 

### Actual behaviour

Builder should build and has stopped doing travels between the construction and his house
- 
Builder asked to report here the issue

### Steps to reproduce the problem

1. 
I update minecollonies mod today and 1st construction is a House update to level 4
2. 
3. 
It asked me to repport.
4. ... Profit

### Logs: Latest.log and crashlog at https://gist.github.com/

- latest.log:

```
[20avr.2020 19:47:26.039] [Server thread/ERROR] [minecolonies/]: Block causes NPE: Block{minecraft:potted_poppy}
java.lang.Exception: null
	at com.minecolonies.coremod.entity.ai.basic.AbstractEntityAIStructure.decreaseInventory(AbstractEntityAIStructure.java:633) ~[?:0.11.840-BETA]
	at com.minecolonies.coremod.entity.ai.basic.AbstractEntityAIStructure.placeBlockAt(AbstractEntityAIStructure.java:479) ~[?:0.11.840-BETA]
	at com.minecolonies.coremod.entity.ai.basic.AbstractEntityAIStructure.decorationStep(AbstractEntityAIStructure.java:374) ~[?:0.11.840-BETA]
	at com.minecolonies.coremod.entity.ai.basic.AbstractEntityAIStructure.lambda$generateStructureGenerator$3(AbstractEntityAIStructure.java:218) ~[?:0.11.840-BETA]
	at com.minecolonies.api.entity.ai.statemachine.basestatemachine.BasicTransition.getNextState(BasicTransition.java:77) ~[?:0.11.840-BETA]
	at com.minecolonies.api.entity.ai.statemachine.basestatemachine.BasicStateMachine.transitionToNext(BasicStateMachine.java:148) ~[?:0.11.840-BETA]
	at com.minecolonies.api.entity.ai.statemachine.basestatemachine.BasicStateMachine.checkTransition(BasicStateMachine.java:131) ~[?:0.11.840-BETA]
	at com.minecolonies.api.entity.ai.statemachine.tickratestatemachine.TickRateStateMachine.checkTransition(TickRateStateMachine.java:124) ~[?:0.11.840-BETA]
	at com.minecolonies.api.entity.ai.statemachine.tickratestatemachine.TickRateStateMachine.tick(TickRateStateMachine.java:103) ~[?:0.11.840-BETA]
	at com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton.func_75246_d(AbstractAISkeleton.java:133) ~[?:0.11.840-BETA]
	at net.minecraft.entity.ai.goal.PrioritizedGoal.func_75246_d(SourceFile:55) ~[?:?]
	at com.performant.coremod.entity.ai.CustomGoalSelector.func_75774_a(CustomGoalSelector.java:363) ~[?:1.15-2.04m]
	at net.minecraft.entity.MobEntity.func_70626_be(MobEntity.java:595) ~[?:?]
	at net.minecraft.entity.LivingEntity.func_70636_d(LivingEntity.java:2262) ~[?:?]
	at net.minecraft.entity.MobEntity.func_70636_d(MobEntity.java:1307) ~[?:?]
	at net.minecraft.entity.AgeableEntity.func_70636_d(SourceFile:159) ~[?:?]
	at com.minecolonies.coremod.entity.citizen.EntityCitizen.func_70636_d(EntityCitizen.java:1135) ~[?:0.11.840-BETA]
	at net.minecraft.entity.LivingEntity.func_70071_h_(LivingEntity.java:2104) ~[?:?]
	at net.minecraft.entity.MobEntity.func_70071_h_(MobEntity.java:274) ~[?:?]
	at net.minecraft.world.server.ServerWorld.func_217479_a(ServerWorld.java:600) ~[?:?]
	at net.minecraft.world.World.func_217390_a(World.java:536) [?:?]
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:401) [?:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:849) [?:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:330) [?:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:784) [?:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:637) [?:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_221]
```

**Comment by Raycoms** [comment]
Can you test this with the newest alpha?

**Comment by Neo2003** [comment]
Hi,

Using structurize-0.10.302-ALPHA + minecolonies-0.11.858-ALPHA, it seems the problem is solved, I have completed 2 building update without any problem, deliveryman and builder are working again and don't ark anymore to report AI problem.
Builder is still asking one kind of block at the time to the deliveryman, but it works like it did with BETA 821. And compared to 821, I found also many small things were fixed:
- In town hall, the word "Chevalier" (French for Knight) is now displayed properly instead of a java name com.thing.somethingelse....
- Many professions accept not mandatory items and stop asking (paper for teacher, shield for fighters....), by the way Students still ask for paper even if they have some.

So this alpha is a lot better :)
